app/src/androidTest/kotlin/at/bitfire/davdroid/sync/account/AccountsCleanupWorkerTest.kt

```diff
@@ -68,7 +67,8 @@ class AccountsCleanupWorkerTest {
         addressBookAccount = Account("Fancy address book account", addressBookAccountType)
 
         // Make sure there are no address books
-        LocalTestAddressBook.removeAll(context)
+        for (account in accountManager.getAccountsByType(addressBookAccountType))
+            accountManager.removeAccountExplicitly(account)
```

Comment:
Do we need this?

Related the comment above. If we really need it, maybe rather use an assertion or use `assume`?

Reply:
Nope, apparently not. `LocalAddressBookStoreTest` was not removing address book accounts after, but before each of  it's tests. Changed with ( https://github.com/bitfireAT/davx5-ose/pull/1310/commits/8aaa1b93648e811678a1acc2cf91d4f913f2ff9d )